src/components/BuildDetailPage/BuildDetailPage.tsx

```diff
@@ -1,5 +1,325 @@
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.id.title} tooltip={buildEntityAttributes.id.tooltip}>
+              {serviceContainerBuild.data.id}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes['user.username'].title}>
+              <Username text={serviceContainerBuild.data.user.username} length={40} />
+            </AttributesItem>
+
+            {/* Build times */}
+            <AttributesItem title={buildEntityAttributes.submitTime.title}>
+              {serviceContainerBuild.data.submitTime && createDateTime({ date: serviceContainerBuild.data.submitTime }).custom}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.startTime.title}>
+              {(buildStatusData[serviceContainerBuild.data.status as BuildStatuses].progress !== 'FINISHED' ||
+                serviceContainerBuild.data.startTime) && (
+                <OnceBuildStarted buildStatus={serviceContainerBuild.data.status}>
+                  {serviceContainerBuild.data.startTime &&
+                    createDateTime({
+                      date: serviceContainerBuild.data.startTime,
+                      includeDateInCustom:
+                        !serviceContainerBuild.data.submitTime ||
+                        !areDatesEqual(serviceContainerBuild.data.submitTime, serviceContainerBuild.data.startTime),
+                    }).custom}
+                </OnceBuildStarted>
+              )}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.endTime.title}>
+              <OnceBuildIsFinished buildStatus={serviceContainerBuild.data.status}>
+                {serviceContainerBuild.data.endTime && (
+                  <>
+                    {
+                      // end (date)time
+                      createDateTime({
+                        date: serviceContainerBuild.data.endTime,
+                        includeDateInCustom:
+                          (!!serviceContainerBuild.data.startTime &&
+                            !areDatesEqual(serviceContainerBuild.data.startTime, serviceContainerBuild.data.endTime)) ||
+                          (!!serviceContainerBuild.data.submitTime &&
+                            !areDatesEqual(serviceContainerBuild.data.submitTime, serviceContainerBuild.data.endTime)),
+                      }).custom +
+                        // duration
+                        (serviceContainerBuild.data.startTime
+                          ? ` (took ${calculateDuration(
+                              serviceContainerBuild.data.startTime,
+                              serviceContainerBuild.data.endTime
+                            )})`
+                          : '')
+                    }
+                  </>
+                )}
+              </OnceBuildIsFinished>
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
```

Comment:
Each of these ContentBoxes should have its own `GridItem`. Now, there is only one for the whole page.
Also, each of the boxes could have a title, as it is on the Artifact detail page.